addons/zeus/functions/fnc_moduleSimulation.sqf

```diff
@@ -0,0 +1,25 @@
+/*
+ * Author: Fisher
+ * Toggle Simulation on object (runs on server only via module framework).
+ *
+ * Arguments:
+ * 0: The module logic <OBJECT>
+ *
+ * Return Value:
+ * None
+ *
+ * Public: No
+ */
+
+#include "script_component.hpp"
+
+params ["_logic"];
+
+private _object = attachedTo _logic;
+if (isNull _object) then {
+    [LSTRING(NoObjectSelected)] call EFUNC(common,displayTextStructured);
```

Comment:
Not optimal in terms of minimising network traffic, but much better UX for zeus and also these modules are only running when prompted by zeus anyway so it's not a big deal.